bamboo/Dockerfile

```diff
@@ -74,6 +74,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     software-properties-common=0.9* \
     apt-utils=2.0.* \
     maven=3.6.* \
+    gradle \
```

Comment:
Why did you add gradle here?

Reply:
I also think that adding it here as well is not necessary.
Either combine the two additions here or only keep the second one.

Reply:
Changed in 003d7e9.